inst/metadata/advanced_run_options.json

```diff
@@ -0,0 +1,66 @@
+{
+  "label": "Advanced",
+  "description": "Advanced model run options",
+  "controlGroups": [
+    {
+      "label": "Maximum iterations",
+      "controls": [
+        {
+          "name": "include_art",
+          "type": "number",
+          "value": 250,
+          "helpText": "Maximum number of iteration for model run",
+          "required": true
+        }
+      ]
+    },
+    {
+      "label": "Number of simulations",
+      "controls": [
+        {
+          "name": "no_of_samples",
+          "type": "number",
+          "value": 3000,
```

Comment:
How long will 3000 take? Is this a sensible default?

Reply:
This is the default used in shiny90, I'm expecting @jeffeaton to review this list and make amendments.